debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/KafkaConnectIT.java

```diff
@@ -0,0 +1,253 @@
+            "DROP SCHEMA IF EXISTS s2 CASCADE;" +
+            "CREATE SCHEMA s1; " +
+            "CREATE SCHEMA s2; " +
+            "CREATE TABLE s1.a (pk SERIAL, aa integer, PRIMARY KEY(pk));" +
+            "CREATE TABLE s2.a (pk SERIAL, aa integer, bb varchar(20), PRIMARY KEY(pk));" +
+            "CREATE TABLE s1.heartbeat (ts TIMESTAMP WITH TIME ZONE PRIMARY KEY);" +
+            "INSERT INTO s1.heartbeat (ts) VALUES (NOW());";
+    private static final String SETUP_TABLES_STMT = CREATE_TABLES_STMT + INSERT_STMT;
+
+    private static final DockerImageName POSTGRES_DOCKER_IMAGE_NAME = DockerImageName.parse(DEFAULT_DEBEZIUM_POSTGRES_EXAMPLE_IMAGE)
+            .asCompatibleSubstituteFor("postgres");
+
+    private static final org.testcontainers.containers.Network NETWORK = org.testcontainers.containers.Network.newNetwork();
+
+    private static final KafkaContainer KAFKA_CONTAINER = new KafkaContainer(DockerImageName.parse("confluentinc/cp-kafka:5.5.3"))
+            .withNetwork(NETWORK);
+
+    private static final PostgreSQLContainer<?> POSTGRES_CONTAINER = new PostgreSQLContainer<>(POSTGRES_DOCKER_IMAGE_NAME)
+            .withNetwork(NETWORK)
+            .withDatabaseName("postgres")
+            .withUsername("postgres")
+            .withPassword("postgres")
+            .withNetworkAliases("postgres");
+
+    private static final DebeziumContainer DEBEZIUM_CONTAINER = new DebeziumContainer(DockerImageName.parse("debezium/connect-base:1.4"))
+            .dependsOn(KAFKA_CONTAINER)
+            .dependsOn(POSTGRES_CONTAINER)
+            .withLogConsumer(new Slf4jLogConsumer(LOGGER).withSeparateOutputStreams())
+            .withNetwork(NETWORK);
+
+    private String oldContainerPort;
+
+    private boolean setupCompleted = false;
+
+    @BeforeClass
+    public static void initTestSuite() {
+        final String debeziumVersion = System.getProperty("project.version");
+        assumeFalse(
+                "ERROR: Project version could not be identified! Make sure env var \"project.version\" is correctly passed for Maven failsafe plugin in POM.",
+                null == debeziumVersion);
+
+        DEBEZIUM_CONTAINER.withFileSystemBind(
+                "target/debezium-connector-postgres-" + debeziumVersion + "-plugin/debezium-connector-postgres",
+                "/kafka/connect/debezium-connector-postgres", BindMode.READ_ONLY).withKafka(KAFKA_CONTAINER);
+    }
+
+    @Before
+    public void setUp() {
+        assumeTrue(
+                "This test suite will be skipped when the \"assembly\" profile is not active.",
+                "true".equals(System.getProperty("assembly.profile.active")));
+        assumeTrue(
+                "This test suite will be skipped when \"assembly.descriptor\" is not set to \"connector-distribution-test\".",
+                "connector-distribution-test".equals(System.getProperty("assembly.descriptor")));
```

Comment:
We need to figure out if this is okay fpr the CI/CD tests?

Reply:
Can you expand a bit on why we should skip this test on CI?

Reply:
I meant it's only skipped automatically now because the `assembly` profile might not be active and we need to additionally set the `assembly.descriptor` property to the newly added `"connector-distribution-test` assembly profile, see the added `connector-distribution-test.xml` which is an assembly descriptor thatm instead of compiling `zip` and `.tar.gz` files, creates a directory that can be mounted into the Kafka Connect container.

Long sentence short, is it okay to enable the `assembly` profile and set the use the new assembly descriptor?

Reply:
So I wouldn't want us to enable an assembly profile by default, if that's what you mean. But how about we move that test to _debezium-testing-testcontainers_ an make sure we enable the right profile when building the required dependencies on CI. I think we should leave the tests which require deployment into an actual KC instance in a single place.